src/saturn_engine/worker/services/rabbitmq.py

```diff
@@ -2,17 +2,40 @@
 import aio_pika.abc
 
 from saturn_engine.utils.asyncutils import cached_property
+from saturn_engine.utils.log import getLogger
 
 from . import MinimalService
 
 
 class RabbitMQService(MinimalService):
     name = "rabbitmq"
 
+    async def open(self) -> None:
+        self.logger = getLogger(__name__, self)
```

Comment:
I'm not familiar with services, is there a reason why this is done in open and not the constructor?

Reply:
I try to keep using `open` instead of `__init__` which is implemeted by the `BaseService` and might be subject to change.